run_sdk_container

```diff
@@ -77,7 +77,7 @@ if [ -z "$name" ] ; then
     name="flatcar-sdk-${arch}-${docker_sdk_vernum}_os-${docker_os_vernum}"
 fi
 
-stat="$(docker ps --all --no-trunc --filter name="$name" --format '{{.Status}}'\
+stat="$(docker ps --all --no-trunc --filter name="^/$name\$" --format '{{.Status}}'\
```

Comment:
Docker docs are just terrible. I found some [message on docker forum](https://forums.docker.com/t/how-to-filter-docker-ps-by-exact-name/2880/11) about this regexp stuff (the slash after `^` is what confused me).

Reply:
It's likely the same message I found :rofl: 
What's even more confusing is that `--filter name="..."` does not do an exact match by default. I feel this violates [POLA](https://en.wikipedia.org/wiki/Principle_of_least_astonishment), but maybe that's just me.